docs/testing-plan.md

```diff
@@ -0,0 +1,130 @@
+# Goal of testing plan
+Manual testing is expensive and we want to create a testing plan to focus on critical parts of the software while automated tests are being developed. 
+
+Some manual tests may be expensive to automate but we should aim to limit this list as much as possible and create as much test coverage to be confident in our deploys.
+
+When testing manual testing we need to keep in mind still both unit tests and integration tests.
+
+Unit tests: individual units or components of a software are tested. 
+Example: A button. Does it look like the mock up? Does it click event work as expected? 
+
+Integrated tests: is the phase in software testing in which individual software modules are combined and tested as a group.How the user can interact with the application throughout an entire workflow.
+Example: A user navigates to a website. The user logs in. The user changes account name. The user logs out.
```

Comment:
the phase in software testing in which individual software modules are combined and tested as a group. The user can interact with the application throughout an entire workflow.
Example: A user navigates to a website, logs in, changes the account name, and finally logs out.